lib/skate/detours/detours.ex

```diff
@@ -193,36 +193,33 @@ defmodule Skate.Detours.Detours do
   end
 
   @doc """
-  Update or create a detour given a user id & detour id.
+  Update or insert a detour given a user id and a XState Snapshot.
   """
-  def update_or_create_detour_for_user(user_id, uuid, attrs \\ %{}) do
-    user = User.get_by_id!(user_id)
-
-    previous_record = uuid != nil && Skate.Repo.get(Detour, uuid)
-
-    detour =
-      case uuid do
-        nil ->
-          create_detour_for_user(user_id, attrs)
```

Comment:
`create_detour_for_user` isn't being used anymore. What do you think: keep it or ditch it?